content/posts/0.27-release.md

```diff
@@ -0,0 +1,202 @@
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    # image: "<image path/url>" # image path/url
+    # alt: "<alt text>" # alt text
+    # caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+Pyodide v0.27.0 is out. This release was focused on improving the long-term
+stability of Pyodide.
+
+## Welcome Agriya Khetarpal to the Pyodide team
+
+Agriya Khetarpal has joined as a new maintainer. Agriya has been active in the
+Scientific Python area and is a contributor to NumPy, SciPy, and scikit-learn.
+He has already significantly strengthened Pyodide's support for various
+Scientific Python packages.
+
+## Build System Improvements
+
+### Decoupling `pyodide-build` from Pyodide runtime
+
+`pyodide-build` is a tool that builds Python packages to run in Pyodide.
+Previously, the version of `pyodide-build` was strongly coupled to the version
+of Pyodide, meaning that if you wanted to build a package against a specific
+version of Pyodide, you had to use the corresponding version of `pyodide-build`.
+The problem with this approach was that even if we improved the build system,
+downstream users would have to wait for the next Pyodide release to use it.
+
+In this release, we have separated `pyodide-build` from the Pyodide runtime.
+This allows us to develop and release `pyodide-build` independently of Pyodide.
+`pyodide-build` is now developed in [pyodide/pyodide-build](https://github.com/pyodide/pyodide-build).
+You can install it with `pip install pyodide-build`.
+
+Recent versions of `pyodide-build` work with Pyodide 0.26 and higher. For
+example, you can use `pyodide-build` version 0.29.2 to build packages for use
+with Pyodide 0.26.4:
+
+```bash
+pip install pyodide-build==0.29.2
+pyodide xbuildenv install 0.26.4
+pyodide build <...>
+```
+
+### Wheels for new packages
+
+This release includes about twenty new packages, most notably popular data
+science packages PyArrow, Polars, and DuckDB.
+
+Previously, we have built all packages in our CI from within the repository. The
+increasing number of packages with long build times have been a strain on our CI
+resources. For instance, PyArrow takes tens of minutes to build. Now, several of
+these packages are built in a separate repository and managed by the package
+maintainers. In Pyodide 0.28.0,
+[we are planning to unvendor the packages from the Pyodide runtime entirely](https://github.com/pyodide/pyodide/issues/4918).
+This will pave the way to supporting many more packages.
+
+### Update to NumPy 2.0
+
+We have updated NumPy to version 2.0.2. This is a major update that required
+updates to many downstream packages in the scientific Python ecosystem. Please
+see [Ecosystem compatibility with numpy 2.0](https://github.com/numpy/numpy/issues/26191)
+for information about downstream compatibility with NumPy 2.0
```

Comment:
```suggestion
We have updated NumPy to version 2.0.2. This is a major update that required
updates to many downstream packages in the scientific Python ecosystem. Please
see [Ecosystem compatibility with numpy 2.0](https://github.com/numpy/numpy/issues/26191)
for information about downstream compatibility with NumPy 2.0.
```

Also, the word "compatibility" comes up twice, could we reword this sentence?

Reply:
Yeah just say "for more information"

Reply:
Makes sense, addressed in a4b6d372cd53621300a2bbd91095c0fad1a3d4ea.